docs/quick-start.zh.md

```diff
@@ -0,0 +1,264 @@
+---
+标题。部署第一个应用程序
```

Comment:
```suggestion
title: 部署第一个应用程序
```

This is the format of our document framework(docusaurus). Please check after machine translation.